core/src/test/resources/doc/examplesExpected/sequence.json

```diff
@@ -0,0 +1,111 @@
+{
+  "groups": [
+    {
+      "name": "Mnemonic",
+      "category": "Mnemonic",
+      "members": []
+    },
+    {
+      "name": "Nemesis",
+      "category": "Nemesis",
+      "members": []
+    },
+    {
+      "name": "Other",
+      "category": "Other",
+      "members": ["nl.nn.adapterframework.doc.testtarget.examples.sequence.Opaque"]
+    },
+    {
+      "name": "Proton",
+      "category": "Proton",
+      "members": []
+    }
+  ],
+  "elements": [
+    {
+      "name": "Master",
+      "fullName": "nl.nn.adapterframework.doc.testtarget.examples.sequence.Master",
+      "isAbstract": false,
+      "isDeprecated": false,
```

Comment:
Deze boolean velden kunnen 'abstract' en 'deprecated' heten. 
Ook op andere plekken: geen 'is'-prefix gebruiken in JSON.

Reply:
Gedaan.